lang/Português.json

```diff
@@ -15,23 +15,23 @@
             "startup": "Abrindo Google Chrome",
             "login_to_complete": "Por Favor complete o login pela pagina web manualmente pressionando o botão Login novamente.",
             "no_token": "Seu Token de autorização Twitch não foi encontrado",
-            "closed_window": "O aplicativo Goggle Chrome foi fechado , portanto o login não foi efetuado com sucesso ."
+            "closed_window": "O aplicativo Goggle Chrome foi fechado, portanto, o login não foi efetuado com sucesso ."
```

Comment:
I think the following translation is closer to the [original one](https://github.com/DevilXD/TwitchDropsMiner/blob/137f540d62560524a9a511a7793616e6d4478e68/translate.py#L240):

```suggestion
            "closed_window": "O Google Chrome foi fechado antes da autenticação ser concluída."
```